terraform/loadbalancer-backend.tf

```diff
@@ -0,0 +1,87 @@
+resource "aws_lb" "backend_alb" {
```

Comment:
why are we using resource for loadbalancer?

Reply:
had to manage multiple rules so modules seemed confusing